packages/frontend/src/components/Game/GameBoard/Tile/index.jsx

```diff
@@ -2,34 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './style.scss';
 
-const Tile = ({ photo, onFlip, index, faceUp, id }) => {
+const Tile = (tile) => {
   const onClickTile = () => {
-    onFlip({ photo, onFlip, index, id, faceUp });
+    tile.onFlip(tile);
```

Comment:
The `onFlip` function is a prop for the Tile component, but isn't a property  of the tile object, so this syntax is a little confusing. I feel like the Tile component should have two props: tile and onFlip or it should be something like `props.onFlip()`, `props.tile.tileProperty`, etc. or it should destructure the individual props and then drill into them 

```javascript
const Tile = (props) => {
  const { tile, onFlip } = props;
  // tile.id, tile.photo
  // onFlip()
}
```